### PR TITLE
New structure

### PR DESCRIPTION
**Pull request by kdclang**
Mistakenly contains updates from previous PR. @ewpatton believes they will just be re-merged and not a problem.

### CODE REVIEW

**Comment by stezelMIT** [comment]
It says the Branch has conflicts.  I am not exactly sure what can be done about this.

File: css/style.css [hunk 1]
```diff
@@ -181,6 +181,11 @@ a.ui-button:active,
 	text-align: center;
 }
 
+.properties {
+	font-weight: bold; 
+	color: #3399ff;">
```
Comment (ewpatton) [inline css/style.css]: the closing quote and angled bracket here should be removed.

Comment (kdclang) [reply css/style.css]: fixed

**Comment by kdclang** [comment]
squash is ok

**Pull request merged**